rust/flatc/src/lib.rs

```diff
@@ -0,0 +1,316 @@
+/*
+ * Copyright 2018 Google Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+//! # flatc
+//!
+//! A library for build scripts to compile FlatBuffer schemas.
```

Comment:
Again I think we need to point explicitly to the main `flatbuffers` package so that users don't get confused. Maybe a description like: `Companion package to flatbuffers, to generate code at compile time.`